### DISCUSSION TITLE
A newbie's question: how to use gifs in preview version [In a Kotlin Multiplatform project]

### DISCUSSION THREAD

**Discussion by flipwind** [coil-kt/coil#2227] [Q&A]
I'm a new to Compose, and I want to use some gifs to my Kotlin Multiplatform project, but I wonder how should I do this.

Actually, I have 3 questions:
1. What's the ```coil-gif``` preview version num?
2. Where should I import this library in ```build.gradle.kts``` in a Kotlin Multiplatform project?
3. How should I use ```coil-gif``` in my code?

If anyone can help me, I will be deeply grateful to you, thanks a lot!\
Finally, forgive my ignorance if I accidentally asked some stupid questions.

**Comment by colinrtwhite** [comment]
`coil-gif` is only supported on Android.

**Comment by flipwind** [comment]
> `coil-gif` is only supported on Android.

Thanks for ur reply!

btw, are there any other ways to show gifs to users by using basic elements which can control their speed? 🙏
